app/user.go

```diff
@@ -1658,6 +1658,7 @@ func (a *App) SendEmailVerification(user *model.User, newEmail, redirect string)
 	}
 
 	if _, err := a.GetStatus(user.Id); err != nil {
+		mlog.Debug("could not get user status", mlog.Err(err))
```

Comment:
I think this is assuming a not found error. I'd suggest that we check for appError.StatusCode = 404, and only then move forward. If it's anything else, just return the error.

Reply:
Makes sense, I was a bit worried about ignoring the error. At least it would appear at debug level :)